src/components/canrawview/canrawview_p.h

```diff
@@ -20,12 +20,15 @@ class CanRawViewPrivate : public QWidget {
 
         tvModel.setHorizontalHeaderLabels({"time", "id", "dir", "dlc", "data"});
         ui->tv->setModel(&tvModel);
-        ui->tv->setColumnWidth(0, 36);
+        ui->tv->setColumnWidth(0, 45);
```

Comment:
Up until know column sizes were hardcoded with magic numbers. Can you find nicer solution here (probably some default sizes with constant or automatic size adjustment).